oldocs/deploy-on-k3d.md

```diff
@@ -51,9 +51,9 @@ In addition to [pre-reqs](pre-reqs.md), install k3d v5.6.0 (only k3d version tes
     ```
 
 1. Install kubestellar controller and OCM space:
-   We are using nginx ingress with tls passthru.
+   We are using nginx ingress with tls pass through.
```

Comment:
"TLS passthrough" actually seems to be common usage. IMO it would be fine to add "passthrough" to the dictionary. Using "TLS pass-through" would be better than "TLS pass through", IMO.